lib/note-editor/index.tsx

```diff
@@ -161,13 +162,15 @@ export class NoteEditor extends Component {
   }
 }
 
-const mapStateToProps = ({ appState: state, settings }) => ({
+const mapStateToProps = ({ appState: state, settings, ui: { note } }) => ({
   allTags: state.tags,
   filter: state.filter,
   fontSize: settings.fontSize,
   editorMode: state.editorMode,
   isEditorActive: !state.showNavigation,
+  note,
   revision: state.revision,
+  tags: get(note, 'data.tags', []),
```

Comment:
it doesn't look like we ever use `props.tags` - did you try removing them? we reference `props.allTags` but otherwise we get `tags` from `this.props.revision || this.props.note`

Reply:
removed in: https://github.com/Automattic/simplenote-electron/pull/1851/commits/d14a6e9141ecbfc237edf5ec2c2abd50a5c2ad05